site/src/pages/DeploySettingsPage/NotificationsPage/NotificationEvents.tsx

```diff
@@ -0,0 +1,224 @@
+import type { Interpolation, Theme } from "@emotion/react";
+import Button from "@mui/material/Button";
+import Card from "@mui/material/Card";
+import Divider from "@mui/material/Divider";
+import List from "@mui/material/List";
+import ListItem from "@mui/material/ListItem";
+import ListItemText, { listItemTextClasses } from "@mui/material/ListItemText";
+import ToggleButton from "@mui/material/ToggleButton";
+import ToggleButtonGroup from "@mui/material/ToggleButtonGroup";
+import Tooltip from "@mui/material/Tooltip";
+import {
+	type selectTemplatesByGroup,
+	updateNotificationTemplateMethod,
+} from "api/queries/notifications";
+import type { DeploymentValues } from "api/typesGenerated";
+import { Alert } from "components/Alert/Alert";
+import { displaySuccess } from "components/GlobalSnackbar/utils";
+import { Stack } from "components/Stack/Stack";
+import {
+	type NotificationMethod,
+	castNotificationMethod,
+	methodIcons,
+	methodLabels,
+} from "modules/notifications/utils";
+import { type FC, Fragment } from "react";
+import { useMutation, useQueryClient } from "react-query";
+import { docs } from "utils/docs";
+
+type NotificationEventsProps = {
+	defaultMethod: NotificationMethod;
+	availableMethods: NotificationMethod[];
+	templatesByGroup: ReturnType<typeof selectTemplatesByGroup>;
+	deploymentValues: DeploymentValues;
+};
+
+export const NotificationEvents: FC<NotificationEventsProps> = ({
+	defaultMethod,
+	availableMethods,
+	templatesByGroup,
+	deploymentValues,
+}) => {
+	const hasWebhookNotifications = Object.values(templatesByGroup)
+		.flat()
+		.some((t) => t.method === "webhook");
+	const hasEmailNotifications = Object.values(templatesByGroup)
+		.flat()
+		.some((t) => t.method === "smtp");
+
+	return (
+		<Stack spacing={4}>
+			{hasWebhookNotifications &&
+				deploymentValues.notifications?.webhook.endpoint === "" && (
+					<Alert
+						severity="warning"
+						actions={
+							<Button
+								variant="text"
+								size="small"
+								component="a"
+								target="_blank"
+								rel="noreferrer"
+								href={docs("/cli/server#--notifications-webhook-endpoint")}
+							>
+								Read the docs
+							</Button>
+						}
+					>
+						Webhook notifications are enabled, but no endpoint has been
```

Comment:
Can you link to the documentation please?
https://coder.com/docs/admin/notifications#webhook